src/airgap/harness-airgap-images.sh

```diff
@@ -9,32 +9,56 @@ fail_count=0
 declare -a failed_images
 declare -a missing_images
 declare -a verified_images
+debug=false
```

Comment:
Should we enable the debug logging by default. Users can turn it off if needed.

Reply:
We can do that, it will print too much info on terminal. I am printing necessary info without debug flag.